plugins/trace.yaml

```diff
@@ -0,0 +1,80 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: trace
+spec:
+  version: "v0.3.4"
+  homepage: https://github.com/iovisor/kubectl-trace
+  shortDescription: Schedule bpftrace programs on your kubernetes cluster using the kubectl.
+  caveats: |
+    kubectl trace is now installed! 
```

Comment:
we usually reserve `caveats` for software that needs to be installed after plugin is installed.

you should probably already have the following examples in your `--help`?

Reply:
I added them to caveats as suggetsed by @corneliusweig